Cargo.lock

```diff
@@ -482,7 +482,7 @@ source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "33d852cb9b869c2a9b3df2f71a3074817f01e1844f839a144f5fcef059a4eb5d"
 dependencies = [
  "libc",
- "windows-sys 0.59.0",
+ "windows-sys 0.52.0",
```

Comment:
Do you need these downgrades? (Three in the commit)

Reply:
No I don't really mind, I was just curious about the downgrade - it looked like something accidental. But other than that, you're probably right I don't think we need these